python/tvm/expr.py

```diff
@@ -138,7 +138,7 @@ def astype(self, dtype):
         return _make.static_cast(dtype, self)
 
 
-class Expr(NodeBase, ExprOp):
+class Expr(ExprOp, NodeBase):
```

Comment:
This relies on [MRO](https://www.python.org/download/releases/2.3/mro/) to select `__eq__`  from `ExprOp`.